elasticdl/python/ps/servicer.py

```diff
@@ -0,0 +1,38 @@
+from google.protobuf import empty_pb2
+
+from elasticdl.proto import elasticdl_pb2_grpc
+
+
+class PserverServicer(elasticdl_pb2_grpc.PserverServicer):
+    """PS service implementation"""
+
+    def __init__(
+        self,
+        parameters,
+        grads_to_wait,
+        optimizer,
+        lr_staleness_modulation=False,
+        use_async=False,
+    ):
+        self._parameters = parameters
+        self._grads_to_wait = grads_to_wait
+        self._optimizer = optimizer
+        self._lr_staleness_modulation = lr_staleness_modulation
+        self._use_async = use_async
+        self._version = 0
+
+    def pull_variable(self, request, _):
+        # TODO: implement this RPC service
```

Comment:
What does `_` mean here?

Reply:
It seems that there too many `push_*` and `pull_*` functions here. Can we wrap them and use a parameter like `data_type` to differentiate them.

Reply:
This is for rpc service. Need to consistent with the names defined in proto file.

Reply:
`_` should be `context`. As it is not used or even implemented in Python GRPC, `_` is used.